disco_aws_automation/disco_elastigroup.py

```diff
@@ -630,6 +629,9 @@ def _roll_group(self, group_id, batch_percentage=100, grace_period=GROUP_ROLL_TI
         self.spotinst_client.roll_group(group_id, batch_percentage, grace_period, health_check_type)
 
         if wait:
+            # wait for the deploy to appear in list
+            time.sleep(10)
```

Comment:
I kind of don't like blind sleeps, but I guess there's no easy way to repeat the call until we get the deployment we want?